frontend/test/metabase/scenarios/question/snippets.cy.spec.js

```diff
@@ -1,59 +1,96 @@
 import { signInAsNormalUser, restore, modal } from "__support__/cypress";
 
+function _clearAndRecursivelyTypeUsingLabel(label, string) {
```

Comment:
nitpick: I don't think this is recursive, so maybe `clearAndTypeEachCharacterUsingLabel` or `clearAndTypeIterativelyUsingLabel` instead?

Reply:
You're absolutely right.
I was struggling with the naming here, tbh. I like "iteratively". Thanks.